src/core/a-assets.js

```diff
@@ -98,6 +98,9 @@ registerElement('a-asset-item', {
       value: function () {
         var self = this;
         var src = this.getAttribute('src');
+        var responseType = this.getAttribute('responseType');
+        if (responseType === null) responseType = 'text';
```

Comment:
`var responseType = this.getAttribute('response-type') || 'text';`

html attribute case insensitive